### PR TITLE
Improve documentation for some installer task types

### PR DESCRIPTION
**Pull request by tannisroot**

### CODE REVIEW

File: docs/installers.rst [hunk 1]
```diff
@@ -485,14 +485,17 @@ reference a ``file id`` or a path, ``args`` to add command arguments,
 to set the directory to execute the command in (defaults to the install path).
 The command is executed within the Lutris Runtime (resolving most shared
 library dependencies). The file is made executable if necessary, no need to run
-chmodx before. You can also use ``env`` (environment variables), ``exclude_processes`` (space-separated list of processes to exclude from being watched), ``include_processes`` (the opposite of ``exclude_processes``, is used to override Lutris' built-in exclude list) and ``disable_runtime`` (run a process without the Lutris Runtime, useful for running system binaries).
+chmodx before. You can also use ``env`` (environment variables), ``exclude_processes`` (space-separated list of processes to exclude from being monitored when determening if the execute phase finished), ``include_processes`` (the opposite of ``exclude_processes``, is used to override Lutris' built-in monitoring exclusition list) and ``disable_runtime`` (run a process without the Lutris Runtime, useful for running system binaries).
```
Comment (AlexanderRavenheart) [inline docs/installers.rst:488]: Minor typos:
- determening -> determining
- exclusition -> exclusion

File: docs/installers.rst [hunk 2]
```diff
@@ -627,16 +630,26 @@ Currently, the following tasks are implemented:
...
     to the executable), ``prefix`` (optional WINEPREFIX),
-    ``arch`` (optional WINEARCH, required when you created win64 prefix), ``blocking`` (if true, do not run the process in a thread), ``working_dir`` (optional working directory), ``include_processes``  (optional space-separated list of processes to include to
-    being watched)
-    ``exclude_processes`` (optional space-separated list of processes to exclude from
-    being watched), ``env`` (optional environment variables), ``overrides`` (optional DLL overrides).
+    ``arch`` (optional WINEARCH value, by default inherited from the `game:` section, which itself defaults to win64. The value can be set to ``win32`` to run the task in a 32-bit prefix.),
+    ``blocking`` (if true, do not run the process in a thread), 
+    ``description`` (a message be shown to the user during the execution of the task), 
+    ``working_dir`` (optional working directory), 
+    ``exclude_processes`` (optional space-separated list of processes to exclude from being monitored when determening if the execute phase finished),
```
Comment (AlexanderRavenheart) [inline docs/installers.rst:637]: Typo: determening -> determining.

File: docs/installers.rst [hunk 3]
```diff
@@ -627,16 +630,26 @@ Currently, the following tasks are implemented:
...
-    ``arch`` (optional WINEARCH, required when you created win64 prefix), ``blocking`` (if true, do not run the process in a thread), ``working_dir`` (optional working directory), ``include_processes``  (optional space-separated list of processes to include to
-    being watched)
-    ``exclude_processes`` (optional space-separated list of processes to exclude from
-    being watched), ``env`` (optional environment variables), ``overrides`` (optional DLL overrides).
+    ``arch`` (optional WINEARCH value, by default inherited from the `game:` section, which itself defaults to win64. The value can be set to ``win32`` to run the task in a 32-bit prefix.),
+    ``blocking`` (if true, do not run the process in a thread), 
+    ``description`` (a message be shown to the user during the execution of the task), 
+    ``working_dir`` (optional working directory), 
+    ``exclude_processes`` (optional space-separated list of processes to exclude from being monitored when determening if the execute phase finished), 
+    ``include_processes`` (the opposite of ``exclude_processes``, is used to override Lutris' built-in monitoring exclusition list),
```
Comment (AlexanderRavenheart) [inline docs/installers.rst:638]: Typo: exclusition -> exclusion

Comment (tannisroot) [reply docs/installers.rst:638]: God, I really shouldn't have merged this half awake

**Pull request merged**